### PR TITLE
Add more contributors as reviewers

### PR DESCRIPTION
**Pull request by timebertt**
**How to categorize this PR?**

/area open-source
/kind enhancement

**What this PR does / why we need it**:

We want to encourage more of our contributors to review PRs, especially in areas where they are more knowledgable than the current set of reviewers/approvers.
We often explicitly invite them to review PRs in their areas of expertise. So their feedback in form of `/lgtm` should also be counted as a successful review for fulfilling the auto-merge requirements by tide, instead of the bot telling them they are not [allowed to add LGTM](https://github.com/gardener/gardener/pull/5934#issuecomment-1118711843).
The list of reviewers can of course be augmented/adapted according to the team/contributor situation.

cc @oliver-goetz @abdasgupta @danielfoehrKn @DockToFuture @istvanballok @Kristian-ZH @ScheererJ @shreyas-s-rao @vlvasilev @voelzmo @vpnachev @wyb1 @petersutter

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -7,11 +7,23 @@ aliases:
...
+  - Kristian-ZH
+  - ScheererJ
+  - shreyas-s-rao
+  - vlvasilev
+  - voelzmo
+  - vpnachev
+  - wyb1
   gardener-approvers:
   - acumino
   - ary1992
```
Comment (rfranzke) [inline OWNERS_ALIASES:30]: Drop  `BeckerMax` from approvers

Comment (rfranzke) [reply OWNERS_ALIASES:30]: And maybe add `petersutter` ?

**Comment by voelzmo** [comment]
LGTM. Would review, but can't! ;)

File: OWNERS_ALIASES (hunk 1 above)
Comment (timebertt) [reply OWNERS_ALIASES:30]: Good call, done :) 
@petersutter welcome to the party 😄

**Review by rfranzke — COMMENTED** [review]
/lgtm

**Comment by vpnachev** [comment]
Is it desired to also be automatically selected as reviewer by prow?
For example https://github.com/gardener/gardener/pull/6023, I was not explicitly invited. And even after I have removed myself from the reviewers, I got requested again by prow.
cc @oliver-goetz

**Comment by shafeeqes** [comment]
This is due to `blunderbuss` plugin in prow, which requests review from people who have had the most changes in the files in the PR. I also feel we should disable it.

**Comment by oliver-goetz** [comment]
Blunderbuss adds reviewers when a PR is created and when reviews are requested. The later case assigned you again to #6023 @vpnachev 

In general I'm not quite sure yet what we want to achieve in the end.
We extended the reviewers, that some people we regularly invite to review PRs are able to /lgtm.
However, I'm not sure whether we would like to request reviews from people outside the gardener core team automatically.

**Pull request merged**